ibmcloud/terraform/cluster/ansible/kube-playbook.yml

```diff
@@ -0,0 +1,270 @@
+
+        if ! grep -q '^PATH=/usr/local/go/bin:\$PATH$' /root/.bashrc; then
+          echo 'PATH=/usr/local/go/bin:$PATH' >> /root/.bashrc
+        fi
+      args:
+        executable: /bin/bash
+        creates: /usr/local/go/bin/go
+
+    - name: Install containerd
+      shell: |
+        set -o errexit -o pipefail
+        containerd_release_tag=$(curl -sL https://api.github.com/repos/containerd/containerd/releases/latest | jq -r .tag_name)
+        rm -fr /tmp/containerd
+        git clone -b CC-main https://github.com/confidential-containers/containerd.git /tmp/containerd
+        (cd /tmp/containerd && make && make install)
+        rm -fr /tmp/containerd
+      environment:
+        PATH: /usr/local/go/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin
+      args:
+        executable: /bin/bash
+        creates: /usr/local/bin/containerd
+    
+    - name: Ensure /etc/containerd directory exists  specific to config
+      file:
+        path: /etc/containerd
+        state: directory
+
+    - name: Copy containerd config file
+      copy:
+        dest: /etc/containerd/config.toml
+        content: |
+          version = 2
+          root = "/var/lib/containerd" 
+          state = "/run/containerd"
+          oom_score = -999
+
+          [grpc]
+            address = "/run/containerd/containerd.sock"
+            uid = 0
+            gid = 0
+
+          [debug]
+            address = "/run/containerd/debug.sock"
+            uid = 0
+            gid = 0
+            level = "debug"
+
+          [plugins]
+            [plugins."io.containerd.runtime.v1.linux"]
+              shim_debug = true
+            [plugins."io.containerd.grpc.v1.cri"]
+              [plugins."io.containerd.grpc.v1.cri".containerd]
+                default_runtime_name = "runc"
+                [plugins."io.containerd.grpc.v1.cri".containerd.runtimes]
+                  [plugins."io.containerd.grpc.v1.cri".containerd.runtimes.runc]
+                    runtime_type = "io.containerd.runc.v2"
+                  [plugins."io.containerd.grpc.v1.cri".containerd.runtimes.kata]
+                    runtime_type = "io.containerd.kata.v2"
+                    cri_handler = "cc"
+
+    - name: Copy systemd unit file for containerd
+      copy:
+        dest: /etc/systemd/system/containerd.service
+        content: |
+          [Unit]
+          Description=containerd container runtime
+          Documentation=https://containerd.io
+          After=network.target
+
+          [Service]
+          ExecStartPre=-/sbin/modprobe overlay
+          ExecStart=/usr/local/bin/containerd --config /etc/containerd/config.toml --log-level debug
+          Delegate=yes
+          KillMode=process
+
+          [Install]
+          WantedBy=multi-user.target
+
+    - name: Start containerd service
+      systemd:
+        name: containerd.service
+        state: started
+    
+    - name: Copy systemd unit file for fix-resolvconf
```

Comment:
We should test if the playbooks work when the Copy systemd unit file for fix-resolvconf and Start fix-resolveconf service tasks are only in the kata playbook

Reply:
Tested and successfully reran Terraform template.
Remove Copy systemd unit file for fix-resolvconf and Start fix-resolvconf service  (which uses systemd) from Kube-playbook